client/helpers/getIcon.js

```diff
@@ -11,6 +11,10 @@ import {
   Organization
 } from 'grommet-icons'
 
+/**
+ * Selects a random icon
+ * @returns - grommet icon component
```

Comment:
```suggestion
 * @returns {import('grommet-icons').Icon} grommet icon component
```

This one's a bit obscure and technically it's non-standard, but some IDEs understand it.

Reply:
Awesome thank you.... I wasn't sure how to go about documenting the returns component! 🧐